src/app/pages/WelcomePage/welcomePage.tsx

```diff
@@ -50,7 +53,15 @@ export default function WelcomePage() {
               fill="#8C0327"
             />
           </svg>
-          Logout
+          <nav>
+            <Link
+              className={styles.link}
+              onClick={() => localStorage.removeItem("Current User")}
```

Comment:
What will happen if the user just closes the tab? Maybe you can use sessionStorage instead? https://developer.mozilla.org/de/docs/Web/API/Window/sessionStorage

Reply:
Thanks for reviewing 🚀  My intention is to keep it simple, as it is an app children shall easily deal with. So I wanted to keep the user logged in, even after the tab is closed. The user should actively log out to change accounts.